scripts/release-validation/diff-with-released.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+ROOT_DIR=../..
```

Comment:
Do we want this to be something like:
```
SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd)"
ROOT_DIR="${SCRIPT_DIR}/../.."
```
so that it can be ran from anywhere, instead of requiring the user to navigate to the scripts directory?